### PR TITLE
guile: add page

### PR DESCRIPTION
**Pull request by arunisaac**
I have added a tldr page for guile, the scheme interpreter.

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/guile.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# guile
+
+> Guile Scheme interpreter.
+
+- Start the Guile Scheme REPL:
+
+`guile`
+
+- Execute script in a given Scheme file:
```
Comment (sbrl) [inline pages/common/guile.md]: How about this?

```suggestion
- Execute the script in a given Scheme file:
```

File: pages/common/guile.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`guile {{script.scm}}`
+
+- Execute a Scheme expression:
+
+`guile -c "{{expression}}"`
+
+- Listen on a port or socket (if the port argument is unspecified, the default is port 37146) for remote REPL connections:
```
Comment (sbrl) [inline pages/common/guile.md]: Perhaps

```suggestion
- Listen on a port or socket (the default port is 37146) for remote REPL connections:
```

...will read better?

**Comment by arunisaac** [comment]
I have implemented the suggested changes.

File: pages/common/guile.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Execute the script in a given Scheme file:
+
+`guile {{script.scm}}`
+
+- Execute a Scheme expression:
+
+`guile -c "{{expression}}"`
+
+- Listen on a port or socket (the default port is 37146) for remote REPL connections:
```
Comment (agnivade) [inline pages/common/guile.md]: socket - do you mean unix domain sockets ?

**Comment by arunisaac** [comment]
I've reworded it a bit. Is it clearer now?

**Pull request merged**